ideascube/mediacenter/templates/mediacenter/document_detail.html

```diff
@@ -19,10 +19,19 @@
 {% endblock twothird %}
 {% block third %}
     {% if user.is_staff %}
-        <ul class="card tinted admin">
-            <li>{% fa 'pencil' 'fa-fw' %}<a href="{% url 'mediacenter:document_update' pk=document.pk %}">{% trans 'Edit' %}</a></li>
-            <li>{% fa 'trash' 'fa-fw' %}<a href="{% url 'mediacenter:document_delete' pk=document.pk %}">{% trans 'Delete' %}</a></li>
-        </ul>
+        {% if document.package_id %}
+            <span class="card admin">
+               {% blocktrans %}
+                   Cannot edit or delete this document.
+                   It is installed by the {{ document.package_id }} package.
+               {% endblocktrans %}
```

Comment:
We are going to a situation where we have two separate application : 
- One displaying the content (so, without modification)
- One allowing user to create/modify content (by creating packages)

It could be interesting on the architecture point of view, to think on how the two applications will interact and how the user will use them. (but it's out of scope for now :) )